### PR TITLE
Update filenames for Assets to match Docupipe generated documentationn

### PR DESCRIPTION
**Pull request by atangOSIsoft**

### CODE REVIEW

File: content/api-reference/assets/toc.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 - name: Assets
-  href: assets-api.md
+  href: assets-asset.md
 - name: Asset types 
-  href: asset-type.md
+  href: assets-asset-type..md
```
Comment (osisoft-mbishop) [inline content/api-reference/assets/toc.yml:4]: Extra period?

```suggestion
  href: assets-asset-type.md
```

Comment (atangOSIsoft) [reply content/api-reference/assets/toc.yml:4]: Ugh, I goofed up. Thanks for catching this!

Comment (atangOSIsoft) [reply content/api-reference/assets/toc.yml]: Accepted suggested change.

Comment (atangOSIsoft) [reply content/api-reference/assets/toc.yml]: Thanks! Will check in! Thanks!

**Pull request merged**